rllib/algorithms/marwil/marwil.py

```diff
@@ -283,13 +277,27 @@ def build_learner_connector(
             device=device,
         )
 
+        # Before anything, add one ts to each episode (and record this in the loss
```

Comment:
I like the Learner connector Pipeline more and more every day. It's super super powerful. We just have to do a good job now documenting this (on me) and making sure users know how to debug into it (by simply switching off tune and setting `num_learners=0`).
🙌